Program.cs

```diff
@@ -3,17 +3,52 @@
+// Valid word conditions - Each word is at least 4 letters long, Each word uses only the 7 seed letters, Each word MUST use the first letter (in this case U),Any letter can be used more than once.
+// Score conditions - 4 letter words score 1 point, Longer words score their no. of letters as points, Any word that uses all 7 seed letters (called pangrams) gets an additional 7 bonus points.
+// Output is a list of words sorted in descending order of score. If two words have the same score, they are displayed in alphabetical order of the words.
+// Pangrams are displayed in green color
 // ---------------------------------------------------------------------------------------
 using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Spelling Bee</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method reads the text file, displays valid words in the arrangement of (score. word) and displays pangrams in green color</summary>
+      static void Main () {
+         string[] words = File.ReadAllLines ("C:words.txt.crdownload");
+         char[] letters = { 'U', 'X', 'A', 'L', 'T', 'N', 'E' };
+         int pangramCount = 0;
+         var eligibleWords = EligibleWords (words, letters, ref pangramCount);
+         ForegroundColor = ConsoleColor.Green;
+         for (int i = 0; i < eligibleWords.Count; i++) {
+            if (i + 1 <= pangramCount) { //Since index is from 0, (i + 1) is checked with pangramCount
+               ForegroundColor = ConsoleColor.Green;
+               Console.WriteLine ($"{eligibleWords[i].Item1,3}. {eligibleWords[i].Item2}"); //If pangram, the word is displayed in green.
+               Console.ResetColor ();
+            } else Console.WriteLine ($"{eligibleWords[i].Item1,3}. {eligibleWords[i].Item2}");
+         }
+         Console.WriteLine ($"----\n{eligibleWords.Sum (x => x.Item1)} total"); // Displays the total points
+      }
+
+      /// <summary>This method checks valid words, find their points and counts the no. of pangrams</summary>
+      /// <param name="stArr">Words Array</param>
+      /// <param name="chArr">Letters Array</param>
+      /// <param name="a">Pangram Count</param>
+      /// <returns>This method returns all valid words and their points in the form of list of tuples</returns>
+      static List<(int, string)> EligibleWords (string[] stArr, char[] chArr, ref int a) {
+         List<(int, string)> list = new ();
+         foreach (string word in stArr) {
+            if (word.Length >= 4 && word.Any (x => x == chArr[0]) && word.All (x => chArr.Any (c => c == x))) { //Checks if letters in word >= 4, must use character 'U' is present, all other letters are in char[] letters.

```

Comment:
As discussed in person, modify the program to have separate functions for validating the words and score computation.